action.yml

```diff
@@ -42,12 +42,13 @@ runs:
         directory: ${{ inputs.directory }}
         path: ${{ inputs.path }}
     - name: Upload Archive
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4.3.0
       with:
         name: ${{ inputs.name }}
         retention-days: ${{ inputs.retention-days }}
         path: ${{ inputs.directory }}/${{ steps.generate-name.outputs.zip_name }}
         if-no-files-found: error
+        overwrite: true
```

Comment:
I don't agree with the overwrite flag, this will delete whatever has already been uploaded with confirmation.
This invites that people will reuse the same artifact name and then wonder why parts of their files are missing